tools/github_actions_test.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -ef
```

Comment:
why '-e' and '-f' ?
also we could use '-x' to avoid the need for the 'echo' -- and the printed command might get out of sync with what is actually executed below